src/ClassAliasAutoloader.php

```diff
@@ -45,21 +66,14 @@ public static function register(Shell $shell, $classMapPath)
     public function __construct(Shell $shell, $classMapPath)
     {
         $this->shell = $shell;
-
-        $vendorPath = dirname(dirname($classMapPath));
+        $this->vendorPath = dirname(dirname($classMapPath));
+        $this->includedAliases = collect(config('tinker.alias', []));
+        $this->excludedAliases = collect(config('tinker.dont_alias', []));
```

Comment:
Since you're not using these properties anywhere else in the class it probably makes more sense to set these as variables inside the `isAliasable` method and not as properties.